### PR TITLE
rtmp-services: Update Vaughn Live and Breakers ingests

### PR DESCRIPTION
**Pull request by invalidtask**
### Description

Updating list of ingest servers for Vaughn Live and Breakers. Also removed recommended settings. I'm not familar with github as I do not use any git stuff, so when I was told in PR #8174 to "force push," I have zero idea what that means or how to go about it. But I do know how to make a pull request. I apologize if this is done incorrectly, however. 

### Motivation and Context

Old ingest servers were listed and new ones were missing. 

### How Has This Been Tested?

Json file in OBS 

### Types of changes

- Tweak (non-breaking change to improve existing functionality) 

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ x ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [ x ] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

**Review by Fenrirthviti — APPROVED** [review]
Approving only from a content standpoint. Commits and commit messages will need to be fixed before this is merged.

**Review by RytoEX — APPROVED** [review]
I've fixed the issues with this PR manually.

In the future:
1. Do not submit a PR from your fork's master branch.
2. Use a single commit to update both `package.json` and `services.json`.
3. Do not add additional commits to make more changes. Amend your commit, then force push to your remote git repo.

If you do not follow this advice, your future PRs may take longer to address or may be closed.

**Comment by invalidtask** [comment]
Noted. As I said, I am not familiar with git beyond a couple basic features. It's not something I utilize in my development process. I apologize and will attempt to implement the suggestions for future PRs (if any.) 

Thank you!

**Pull request merged**
